charts/datadog/README.md.gotmpl

```diff
@@ -68,27 +72,41 @@ Now, the installation command contains the reference to the secret.
 
 ```bash
 helm install --name <RELEASE_NAME> \
-  --set datadog.apiKeyExistingSecret=$DATADOG_SECRET_NAME datadog/datadog
+  --set datadog.apiKeyExistingSecret=$DATADOG_API_SECRET_NAME datadog/datadog
 ```
 
-**Note**: Provide a secret for the application key (AppKey) using the `datadog.appKeyExistingSecret` chart variable.
-
 ### Enabling the Datadog Cluster Agent
 
+The Datadog Cluster Agent is now enabled by default.
+
 Read about the Datadog Cluster Agent in the [official documentation](https://docs.datadoghq.com/agent/kubernetes/cluster/).
 
```

Comment:
Should we just completely drop this paragraph ?

Reply:
For the paragraph I kept it for now. because it might be good to keep a link to the cluster-agent doc
and we have some sub section